app/scripts/modules/core/src/pipeline/status/Artifact.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { ShallowWrapper, shallow } from 'enzyme';
+import { mock } from 'angular';
+import { REACT_MODULE } from 'core/reactShims';
```

Comment:
Typically third-party imports, core imports, and local-module imports are newline-separated per this hard-to-find conventions doc: https://github.com/spinnaker/deck/wiki/Conventions#imports

Reply:
Thanks!  I had no idea about those conventions---I fixed all the imports I touch to follow them